docs/_static_html/conf/eu/2016/cfp/index.html

```diff
@@ -114,7 +114,7 @@
 
     
   <div class="hero hero-conf">
-    <div class="hero-logo">
+    <div class="hero-logo" data-proofer-ignore>
       <a href="/conf/eu/2016/">
         <img src="../../../../_static/img/2016/site-logo.png" />
       </a>
```

Comment:
Would the error go away if the image had an `alt` attribute? It seems like it's an empty link for people who don't load images, which isn't ideal for accessibility. Though I doubt there's many people going back to look at the logo from 6 years ago.

Reply:
Ah, I was over enthusiastic in my search replace. The ignore there is intended for the bunch of `a` tags that don't have a reference, like this one: https://github.com/writethedocs/www/pull/1764/files#diff-7a114b4493925c9b9c38c932134ded5be5f69f6f371dec742f42271428f9d96cR45

We don't check for empty or missing alt tags, because we have so many, although I'd definitely be interested in fixing that, at least in current conferences. What is best practice for alt tags for images that don't mean anything and are just part of the theme?

Reply:
For those, an empty alt attribute:

```html
<img src="image.png" alt="" />
```
That signals that it's just decoration and is intentionally blank